test/pylib/scylla_cluster.py

```diff
@@ -989,6 +990,15 @@ async def handle_join_failure():
                           ip_addr, workdir, str(exc))
             await handle_join_failure()
             raise
+        except:
+            # catches everything
+            workdir = '<unknown>' if server is None else server.workdir.name
+            exc_type, exc_value, _ = sys.exc_info()
+            self.logger.error("Failed to start Scylla server at host %s in %s: %s(%s)",
+                              ip_addr, workdir, exc_type.__name__, exc_value)
+            self.logger.error("Traceback: ", exc_info=sys.exc_info())
```

Comment:
Here too.

Reply:
`exc_info` keyword argument is specific to logger, see https://docs.python.org/3/library/logging.html#logging.Logger.debug . i am quoting the related paragraph here:

> If exc_info does not evaluate as false, it causes exception information to be added to the logging message. If an exception tuple (in the format returned by [sys.exc_info()](https://docs.python.org/3/library/sys.html#sys.exc_info)) or an exception instance is provided, it is used; otherwise, [sys.exc_info()](https://docs.python.org/3/library/sys.html#sys.exc_info) is called to get the exception information.

it's very handy in this case.